webapp/src/shared/components/Game/hooks/useOnlineGame.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import socket from '../../../socket';
+import { GameCommand, GameState } from 'pong-engine';
+import { WsException } from '../../../types';
+import { useNotificationContext } from '../../../context/NotificationContext';
+
+// Tipar mensajes ws -> investigar si se puede hacer con OpenApi
```

Comment:
I don't think OpenApi does this.
But we can always have some more types added to a shared package (pong-engine in this case).

Reply:
Yes, @gbudau told me about a lib we could use but we’re not 100% sure if it worth